.github/workflows/docker_publish.yml

```diff
@@ -182,56 +48,19 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Build and push Dockerfile_3_moab
-        uses: docker/build-push-action@v2
-        with:
-          file: CI/Dockerfile
-          target: moab
-          context: .
-          push: true
-          build-args: |
-            OWNER=${{ github.repository_owner }}
-            TAG=ci_testing
-            UBUNTU_VERSION=${{ matrix.ubuntu_versions }}
-            COMPILER=${{ matrix.compiler }}
-            HDF5=${{ matrix.hdf5_versions }}
-            MOAB=${{ matrix.moab_versions }}
-          tags: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}:ci_testing
-
-  housekeeping:
-    if: ${{ github.repository_owner != 'svalinn' || github.event_name == 'pull_request' }}
-    needs: [build-housekeeping-img]
-    runs-on: ubuntu-latest
-    env:
-      hdf5_versions: 1.10.4
-      hdf5_build_dir: hdf5_build_dir    
-    strategy:
-      matrix:
-        ubuntu_versions : [18.04,]
-    container: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:ci_testing
-
-    name: Running Housekeeping scrips
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
+      - name: Installing Dependencies in Docker image
+        uses: firehed/multistage-docker-build-action@v1
         with:
-          submodules: recursive
-
-      - name: Setup
-        run: |
-          git config --global --add safe.directory ${GITHUB_WORKSPACE}
-          mkdir /root/build_dir/
-          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
-          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
-          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
+          repository: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}
+          stages: base, external_deps, hdf5
+          server-stage: moab
+          quiet: false
+          tag-latest-on-default: false
+          dockerfile: CI/Dockerfile
+          build-args: OWNER=${{ github.repository_owner }}, TAG=ci_testing, UBUNTU_VERSION=${{ matrix.ubuntu_versions }}, COMPILER=${{ matrix.compiler }}, HDF5=${{ matrix.hdf5_versions }}, MOAB=${{ matrix.moab_versions }}, build_mw_reg_tests=ON
```

Comment:
I don't think we use the `TAG` build-arg anywhere

Reply:
resolved by removing both of these build args from the command